src/rpc/mempool.cpp

```diff
@@ -1082,10 +1082,15 @@ static RPCHelpMan submitpackage()
             for (const auto& tx : txns) {
                 UniValue result_inner{UniValue::VOBJ};
                 result_inner.pushKV("txid", tx->GetHash().GetHex());
+                const auto wtxid_hex = tx->GetWitnessHash().GetHex();
```

Comment:
Fwiw not a huge deal but I wouldn't recommend doing an unnecessary refactor at the same time as a fix to the missing RPC result.

Looks like this is creating a copy of the string to avoid 2 calls to `GetHex()`, which seems sensible.

Reply:
Apologies if it was too minor to refactor! Will try to keep it more surgical in the future for these types of touch-up PRs.